code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -346,7 +346,7 @@
 
 /obj/item/gun/ballistic/automatic/sniper_rifle/syndicate
 	name = "syndicate sniper rifle"
-	desc = "An illegally modified .50 cal sniper rifle with suppression compatibility. Quickscoping still doesn't work."
+	desc = "An illegally modified .50 BMG anti-materiel rifle with suppression compatibility. Quickscoping still doesn't work, and the suppressor probably won't do much either."
```

Comment:
i was gonna say isn't it anti-material, but it isn't. wtf.

Reply:
English is a funny language.

Reply:
One of the many mysteries.

Reply:
Materiel refers to military materials/equipment specifically

Reply:
You can blame the french for this one